Running in wrapper mode for '70'

Running python job.

[name: "CPU-QUOTA"
int_val: 95
int_val: 35
int_val: 81
int_val: 85
int_val: 72
int_val: 5
int_val: 43
, name: "DISK"
int_val: 36
int_val: 38
int_val: 22
int_val: 8
int_val: 20
int_val: 21
int_val: 38
, name: "MEMORY"
int_val: 9
int_val: 12
int_val: 25
int_val: 39
int_val: 14
int_val: 27
int_val: 25
, name: "NET"
int_val: 13
int_val: 9
int_val: 24
int_val: 36
int_val: 31
int_val: 17
int_val: 15
]
The paramters are {u'CPU-QUOTA': array([95, 35, 81, 85, 72,  5, 43]), u'NET': array([13,  9, 24, 36, 31, 17, 15]), u'DISK': array([36, 38, 22,  8, 20, 21, 38]), u'MEMORY': array([ 9, 12, 25, 39, 14, 27, 25])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
----------------------------------------
Problem running the job:
(<class 'paramiko.ssh_exception.BadAuthenticationType'>, BadAuthenticationType('Bad authentication type', [u'publickey']), <traceback object at 0x7fd41b74e440>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 147, in explore_spearmint
    client = re.get_client(machine)
  File "/home/ubuntu/throttlebot/src/remote_execution.py", line 6, in get_client
    client.connect(ip, username="quilt", password="")
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 424, in connect
    passphrase,
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 714, in _auth
    raise saved_exception
BadAuthenticationType: ('Bad authentication type', [u'publickey']) (allowed_types=[u'publickey'])
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 0.44 seconds.
